src/pybind/mgr/prometheus/module.py

```diff
@@ -594,6 +594,12 @@ class Module(MgrModule):
             min=400,
             max=599,
             runtime=True
+        ),
+        Option(
+            name='fetch_perf_counters_metrics',
+            type='bool',
+            default=False,
+            runtime=True
```

Comment:
Probably worth to change the docs to let users know about the change

Reply:
> Probably worth to change the docs to let users know about the change

Yes, it'll be addressed along with exporter documentation https://tracker.ceph.com/issues/58163

Reply:
new option needed in : 
https://docs.ceph.com/en/quincy/mgr/prometheus/#configuration.

I think it is more coherent to add the documentation in this PR.

Reply:
I've now added the documentation as well!